dash/dash.py

```diff
@@ -19,6 +19,7 @@
 import flask
 from flask import Flask, Response
 from flask_compress import Compress
+from werkzeug.debug import get_current_traceback
```

Comment:
minor: 
```suggestion
from werkzeug.debug.tbtools import get_current_traceback
```

Reply:
Right, that's where it's actually defined... but is that better? I thought it'd be safest to import from as high up in the package as possible, in case the package internals get restructured later. That said it looks like even internally to werkzeug they generally use the import you suggest, and I guess `werkzeug.debug.__init__` isn't really importing things to export a clean namespace, it just wants to use `get_current_traceback` itself. OK, I'll change it.